worker/src/lib/dbLogger/logResponse.ts

```diff
@@ -200,12 +101,12 @@ export async function logRequest(
       id: request.requestId,
       path: request.path,
       body: request.omitLog ? {} : unsupportedImage(requestBody),
-      auth_hash: request.providerApiKeyAuthHash,
+      auth_hash: "",
```

Comment:
I'm confused

Reply:
We should honestly remove the auth_hash column it is never used